keylime-push-model-agent/src/context_info_handler.rs

```diff
@@ -101,16 +102,15 @@ mod tests {
     }
 
     #[tokio::test]
-    #[cfg(feature = "testing")]
```

Comment:
Why did you remove this "testing" feature label? If I am not wrong, all the TPM related tests are executed with it